index.js

```diff
@@ -31,6 +31,7 @@ class Client {
     this.serverless = serverless;
     this.provider = 'aws';
     this.aws = this.serverless.getProvider(this.provider);
+	this.options = options;
```

Comment:
Can you remove the tabs and add in spaces everywhere? You can review in GitHub to compare the spacing here.